shared/js/background/url-parameters.js

```diff
@@ -65,34 +45,20 @@ function stripTrackingParameters (url) {
         return parametersRemoved
     }
 
-    // Remove any plain-text tracking parameters first, since they are cheaper
-    // to remove.
-    for (const key of plainTextTrackingParameters) {
-        if (url.searchParams.has(key)) {
-            url.searchParams.delete(key)
+    // Remove tracking parameters
+    // Note: We can't use URLSearchParams here because of issues with
```

Comment:
```suggestion
    // Note: We can't use url.searchParams here because adding/removing parameters
    //            with URLSearchParams adjusts the encoding of the other parameters.
    //            See https://url.spec.whatwg.org/#urlsearchparams
    // 
```